ibis/backends/tests/test_udf.py

```diff
@@ -174,10 +184,11 @@ def add_one_pyarrow(s: int) -> int:  # s is series, int is the element type
             add_one_pyarrow,
             marks=[
                 mark.notyet(
-                    ["snowflake", "sqlite", "pyspark", "flink"],
+                    ["snowflake", "sqlite", "flink"],
                     raises=NotImplementedError,
                     reason="backend doesn't support pyarrow UDFs",
-                )
+                ),
+                pyspark_version_mismatch,
```

Comment:
We actually have as special marker for this situation, there are plenty of other backends where this kind of problem comes up :)

```suggestion
                mark.xfail_version(pyspark=["pyspark<3.5"]),
```